src/pages/index.tsx

```diff
@@ -1,16 +1,23 @@
 import React, { FunctionComponent } from 'react';
-import H from '../atoms/H';
 import Layout from '../organisms/Layout';
-import P from '../atoms/P';
-import B from '../atoms/B';
-
+import ArticleCard from '../molecules/ArticleCard';
 const Index: FunctionComponent = () => (
-    <Layout>
-      <H>Main content</H>
-      <P>
-        Hej <B>bold</B>
-      </P>
-    </Layout>
+  <Layout>
+    <ArticleCard
```

Comment:
För att göra det så smidigt som möjligt att koppla på sanity sen skulle du kunna lägga allt som ska in i komponenterna i en lista och mappa igenom och skapa komponenterna därifrån